pkg/kube/iscsid.conf

```diff
@@ -0,0 +1,337 @@
+#
```

Comment:
Is this an imported file from somewhere? If so it might make sense to put a comment here saying "From XXX".

Reply:
Fixed